### ISSUE TITLE
WooCommerce Paypal Payments plugin does not work with WooCommerce Blocks checkout plugin

### ISSUE THREAD

**Issue by vineetag** [woocommerce/woocommerce-paypal-payments#1216]
### Describe the Bug

I am using WooCommerce blocks for the Gutenberg editor Version 9.6.3. After adding 'checkout' block, the page says the PayPal payments extension is not supported.

PayPal is a ley functionality for our website www.theoffbeatshop.com and not having it supported in WooCommerce blocks is creating an issue.

### Screenshots

<img width="1789" alt="image" src="https://user-images.githubusercontent.com/487893/221795151-a33b9715-0e47-4b7a-979f-ee113eb77c66.png">

### Environment

 - WordPress Version - 6.1.1
 - WooCommerce Version - 7.4.0
 - Plugin Version - 2.0.2
 - Browser [e.g. Chrome, Safari] and Version
 - Any other plugins installed - WooCommerce Blocks Version 9.6.3

### Additional Details
`
### WordPress Environment ###

WordPress address (URL): https://theoffbeatshop.com
Site address (URL): https://theoffbeatshop.com
WC Version: 7.4.0
REST API Version: ✔ 7.4.0
WC Blocks Version: ✔ 9.6.3
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.1.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx/1.22.1
PHP Version: 8.0.25
PHP Post Max Size: 512 MB
PHP Time Limit: 300
PHP Max Input Vars: 5000
cURL Version: 7.83.1
OpenSSL/1.1.1s

SUHOSIN Installed: –
MySQL Version: 5.7.26-29-log
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.4.0
WC Database Prefix: wp_
Total Database Size: 18.37MB
Database Data Size: 12.38MB
Database Index Size: 5.99MB
wp_woocommerce_sessions: Data: 1.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 3.02MB + Index: 1.28MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 1.48MB + Index: 1.83MB + Engine InnoDB
wp_atum_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_atum_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_atum_product_data: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_automations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_automation_runs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_automation_run_logs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_automation_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_automation_versions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletters: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_scheduled_tasks: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_bounces: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscriber_tag: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_tags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_user_agents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 1.11MB + Index: 0.08MB + Engine InnoDB
wp_postmeta: Data: 0.22MB + Index: 0.13MB + Engine InnoDB
wp_posts: Data: 1.06MB + Index: 0.06MB + Engine InnoDB
wp_shortpixel_folders: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shortpixel_meta: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shortpixel_postmeta: Data: 0.08MB + Index: 0.08MB + Engine InnoDB
wp_shortpixel_queue: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_social_users: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yith_wcwl: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yith_wcwl_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 81
atum_supplier: 3
blocks: 13
customize_changeset: 67
featured_item: 8
jp_img_sitemap: 1
jp_sitemap: 1
jp_sitemap_master: 1
mailpoet_page: 1
nav_menu_item: 12
page: 22
post: 15
product: 35
product_variation: 18
revision: 87
shop_coupon: 1
shop_order: 1
wp_global_styles: 1
wpc_product_badge: 1
wpcf7_contact_form: 3

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (17) ###

Akismet Anti-Spam: by Automattic – 5.0.2
ATUM Inventory Management for WooCommerce: by Stock Management Labs™ – 1.9.26.2
Contact Form 7: by Takayuki Miyoshi – 5.7.4
Site Kit by Google: by Google – 1.94.0
Jetpack Boost: by Automattic - Jetpack Site Speed team – 1.7.0
Jetpack: by Automattic – 11.8.4
MailPoet: by MailPoet – 4.7.0
Nextend Social Login: by Nextendweb – 3.1.7
Nextend Social Login Pro Addon: by Nextendweb – 3.1.7
Password Protected: by WPExperts – 2.6.2
ShortPixel Image Optimizer: by ShortPixel - Convert WebP/AVIF & Optimize Images – 5.2.1
WooCommerce Blocks: by Automattic – 9.6.3
WooCommerce Payments: by Automattic – 5.5.0
WooCommerce PayPal Payments: by WooCommerce – 2.0.2
WooCommerce Shipping & Tax: by WooCommerce – 2.2.3
WooCommerce: by Automattic – 7.4.0
YITH WooCommerce Wishlist: by YITH – 3.18.0

### Dropin Plugins (1) ###

object-cache.php: Redis Object Cache Drop-In

### Must Use Plugins (1) ###

EasyWP Plugin: by Namecheap
Inc. – 1.0.1.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
Order datastore: WC_Order_Data_Store_CPT

### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: #509 - /terms/

### Theme ###

Name: Flatsome Child
Version: 3.0
Author URL: 
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.16.6
Parent Theme Author URL: https://uxthemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/rating.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php
flatsome/woocommerce/content-product_cat.php

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: US
WooCommerce currency supported: ✔
Advanced Card Processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: ✔
Vault enabled: ✔
Logging enabled: ✔
Reference Transactions: –
Used PayPal Checkout plugin: –
Tracking enabled: ✔

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
new-product-management-experience
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-03-01 08:24:27 -08:00
Options: ✔
Notes: 67
Onboarding: completed

### WooCommerce Payments ###

Version: 5.5.0
Connected to WPCOM: Yes
Blog ID: 131475487
Account ID: acct_1Mg2OaFlUvS9GADl

### Action Scheduler ###

Canceled: 1
Oldest: 2023-02-10 20:08:22 +0000
Newest: 2023-02-10 20:08:22 +0000

Complete: 6,158
Oldest: 2023-01-28 08:27:59 +0000
Newest: 2023-02-28 08:25:00 +0000

Failed: 9
Oldest: 2022-12-07 20:33:20 +0000
Newest: 2023-02-14 21:34:07 +0000

Pending: 4
Oldest: 2023-02-28 08:25:58 +0000
Newest: 2023-03-01 10:51:33 +0000

### Status report information ###

Generated at: 2023-02-28 00:25:14 -08:00
`
</details>

**Comment by ethanclevenger91** [comment]
This is a duplicate of #1090. However:

The checkout block is in core - not hidden in a feature plugin anymore. A new WooCommerce install also does not include the old PayPal gateway. So a brand new WooCommerce install, using the checkout block included for anyone to use without any "beta" branding, can't use the currently recommended PayPal Payments plugin?

I would have expected that first-party WooCommerce tools would be ready to support the new checkout block, and would really appreciate if the WooCommerce team stopped rolling out beta tools to the general public.

Can we at least get a timetable on this? [The page linked in the other issue](https://woocommerce.com/document/woocommerce-paypal-payments/#faq-does-paypal-payments-support-blocks) says "Support for the new Block Cart & Checkout in development for early 2023" seems to be missing some words or has typos, so it isn't clear what exactly is meant, and clarification would be appreciated.

**Comment by InpsydeNiklas** [comment]
Support for the WooCommerce Blocks is currently in development, and an initial release may be included in the next feature update. 
Status updates will be provided in #211.
If you need to use PayPal with the Block checkout, then the PayPal Standard integration can be [re-enabled with a filter](https://developer.woocommerce.com/2021/07/12/developer-advisory-paypal-standard-will-be-hidden-on-new-installs/).
